circleci/images/extbuilder/files/sbin/install-extdeps

```diff
@@ -30,8 +32,9 @@ APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add -
 
 # add pgdg repo to sources
 echo "Writing /etc/apt/sources.list.d/pgdg.list..." >&2
-echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main" > \
-     /etc/apt/sources.list.d/pgdg.list
+echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main ${pg_latest}" > \
```

Comment:
has this changed in the postgres packaging? we didn't need this before. I would expec this diff to be nothing more then adding `12` to the `pg_majors`